api/VRSConnection.go

```diff
@@ -1,15 +1,53 @@
 package api
 
 import (
+	"errors"
 	"github.com/nuagenetworks/libvrsdk/ovsdb"
 	"github.com/socketplane/libovsdb"
 )
 
+type portNameChannelMap map[string]chan *PortIPv4Info
+
+type portNamePortInfoMap map[string]PortIPv4Info
+
+type Registration struct {
+	Brport   string
+	Channel  chan *PortIPv4Info
+	Register bool
+}
+
 // VRSConnection represent the OVSDB connection to the VRS
 type VRSConnection struct {
-	ovsdbClient *libovsdb.OvsdbClient
-	vmTable     ovsdb.NuageTable
-	portTable   ovsdb.NuageTable
+	ovsdbClient         *libovsdb.OvsdbClient
+	vmTable             ovsdb.NuageTable
+	portTable           ovsdb.NuageTable
+	updatesChan         chan *libovsdb.TableUpdates
+	pncTable            portNameChannelMap
+	pnpTable            portNamePortInfoMap
+	stopChannel         chan bool
+	registrationChannel chan *Registration
+}
+
+// Disconnected will retry connecting to OVSDB
+// and continue to register for OVSDB updates
+func (vrsConnection VRSConnection) Disconnected(ovsClient *libovsdb.OvsdbClient) {
```

Comment:
Do we also need to handle connection to OVSDB lost case here to have retries to connect and restart monitoring Port table?